runtime/manual/node/cjs_to_esm.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Updating from CommonJS to ESM"
+---
```

Comment:
Start the section with "Deno supports modern Node projects out of the box. However if your project uses old-style CommonJS imports (i.e. `require`) you will need to modernize your codebase before using Deno."